akka-cluster-typed/src/main/scala/akka/cluster/ddata/typed/scaladsl/Topic.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.ddata.typed.scaladsl
+import akka.actor.typed.receptionist.{Receptionist, ServiceKey}
+import akka.actor.typed.{ActorRef, Behavior}
+import akka.actor.typed.scaladsl.Behaviors
+import akka.annotation.DoNotInherit
+
+object Topic {
+
+  /**
+   * NOT FOR USER EXTENSION
+   */
+  @DoNotInherit
+  sealed trait TopicCommand
+
+}
+
+class Topic[T](serviceKey: ServiceKey[T]) {
```

Comment:
maybe it belongs to the Routers?

Reply:
Maybe, the initial reason for writing it is deduplication of internode publications, but maybe there are things that makes it useful locally as well

Reply:
Since the purpose is to have one `Topic(key)` per ActorSystem it should perhaps be managed by an extension?